scenes/game/local_game.gd

```diff
@@ -2031,10 +2041,10 @@ class Player:
 							strike_stat_boosts.dodge_at_range_max = -1
 							strike_stat_boosts.dodge_at_range_from_buddy = false
 					"powerup":
-						strike_stat_boosts.power -= effect['amount']
+						add_power_bonus(-effect['amount'])
 					"powerup_both_players":
-						strike_stat_boosts.power -= effect['amount']
-						opposing_player.strike_stat_boosts.power -= effect['amount']
+						add_power_bonus(-effect['amount'])
```

Comment:
won't this not properly update the positive power bonuses (because it's being passed a negative value for positive bonuses)? might be easier to have a separate remove_power_bonus function, or a second parameter

Reply:
oh, yea